.circleci/config.yml

```diff
@@ -10,31 +10,11 @@
 #  See: http://github.com/globality-corp/globality-build
 #
 #
-
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: ${AWS_ECR_DOMAIN}/globality-build:2019.31.2
```

Comment:
Ripped out all of this docker/aws magic to make something easier for newbies or external folks to understand and play around with.